src/App.js

```diff
@@ -1,19 +1,53 @@
 import React from 'react';
+import Tabs from './components/Tabs/Tabs';
 
 import './App.scss';
 
 /* eslint-disable */
-const tabs = [
+const tabsList = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.PureComponent {
+  state = {
+    tabs: [...tabsList],
+    selectedTab: {
+      id: 'tab-1', title: 'Tab 1', content: 'Some text 1',
+    },
+    selectedTabId: 'tab-1',
```

Comment:
виділену табу краще витягувати з масиву, щоб не пропусути тут новий об'єкт, а використати вже існуючий

```suggestion
    selectedTab: tabsList[0],
    selectedTabId: tabsList[0].id,
```